tests/functional/adapter/constraints/test_bigquery_constraints.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+import re
+import json
+from dbt.tests.util import (
+    run_dbt,
+    get_manifest,
+    run_dbt_and_capture
+)
+from dbt.tests.adapter.constraints.test_constraints import (
+    TestModelLevelConstraintsEnabledConfigs,
+    TestModelLevelConstraintsDisabledConfigs,
+    TestSchemaConstraintsEnabledConfigs,
+    TestModelLevelConstraintsErrorMessages
+)
+
+
+my_model_sql = """
+{{
+  config(
+    materialized = "table"
+  )
+}}
+select
+  1 as id,
+  'blue' as color,
+  cast('2019-01-01' as date) as date_day
+"""
+
+my_model_error_sql = """
+{{
+  config(
+    materialized = "table"
+  )
+}}
+select
+  null as id,
+  'blue' as color,
+  cast('2019-01-01' as date) as date_day
+"""
+
+model_schema_yml = """
+version: 2
+models:
+  - name: my_model
+    config:
+      constraints_enabled: true
+    columns:
+      - name: id
+        data_type: integer
+        description: hello
+        constraints: ['not null','primary key']
+        check: (id > 0)
+        tests:
+          - unique
+      - name: color
+        data_type: string
+      - name: date_day
+        data_type: date
+  - name: my_model_error
+    config:
+      constraints_enabled: true
+    columns:
+      - name: id
+        data_type: integer
+        description: hello
+        constraints: ['not null','primary key']
+        check: (id > 0)
+        tests:
+          - unique
+      - name: color
+        data_type: string
+      - name: date_day
+        data_type: date
+"""
+
+database_name = "database_placeholder"
+schema_name = "schema_placeholder"
+
+_expected_sql = f"""
+  create or replace table
+    "{database_name}"."{schema_name}"."my_model"
```

Comment:
This is failing because bigquery uses ` vs. quotes: https://github.com/dbt-labs/dbt-bigquery/actions/runs/3744300377/jobs/6357515726#step:7:571
```suggestion
    `{database_name}`.`{schema_name}`.`my_model`
```